### ISSUE TITLE
Announcement/Tutorial Article

### ISSUE THREAD

**Issue by Romulus10** [fennelLabs/Fennel-Protocol#43]

**Comment by isaacadams** [comment]
where will we be publishing articles?

**Comment by Romulus10** [comment]
Good catch on the title - a couple keys on my keyboard have been a little jank lately. I think it needs cleaning.

Anyway, our main website has a blog set up on GitHub Pages. We're considering between that and something like Medium. The biggest advantage to Medium is that it's a little nicer to look at. The biggest advantage to ours is that we just host it right on our website.

**Comment by isaacadams** [comment]
I recently installed a browser extension which will point out spelling errors -- sometimes, it is a little annoying because I am typically not that picky, but I think it's worth the pestering since it helps me notice errors I would have otherwise not seen but should be corrected.

Medium/GitHub Pages sounds great. I agree with your assessment of the two. Sounds good!

**Comment by Romulus10** [comment]
I kind of like hosting our own blog then announcing posts on Twitter/LinkedIn.

**Comment by isaacadams** [comment]
yes -- possible idea to use both?

GitHub Pages Blog: official articles, feature announcements, in-depth exploration of Fennel concepts, etc.

Medium: less-official articles; community-oriented, community-written (still must be approved by Fennel team or Fennel approved community organizer), etc.

**Comment by PhillyWebGuy** [comment]
Is there something I can start on with this stuff? I might not have all the details yet, but if there's something to read, that can be summarized for the public, that's something I can certainly do.

**Comment by Romulus10** [comment]
I think that this is probably going to be a summary of the grant application through the lens of a discussion of what we've built so far, so I think if you're familiar with the grant app all we need then is to finish documenting the work we've done. In terms of what to read, I guess the first step is to figure out whether someone relatively new to the project can get a node up and running and figure out how to send transactions based on what we have in the README/Wiki. We'll also want to figure out what's missing from that documentation (other than what we're already working on documenting). Once we've got that we can refine it into the article.
